### PR TITLE
docs: add from_cfn section in blogpost v1.25

### PR DESCRIPTION
**Pull request by KollaAdithya**
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the Apache 2.0 License.

### CODE REVIEW

**Review by iamhopaul123 — APPROVED** [review]

File: site/content/blogs/release-v125.en.md [hunk 1]
```diff
@@ -33,6 +33,39 @@ Copilot v1.25 brings several new features and improvements:
 ## Environment Addons
 ### Import Values From CloudFormation Stacks In Workload Manifests
 
+You can now import values from environment addons CloudFormation stack or any other arbitary stack in your workload manifest using `from_cfn`.
+To reference a value from another CloudFormation stack, users should first export the output value from the source stack.
```
Comment (Lou1415926) [inline site/content/blogs/release-v125.en.md:37]: Can we have an example of what the `Export` should look like, and a link to CFN doc that talks about `Export`?

Comment (KollaAdithya) [reply site/content/blogs/release-v125.en.md:37]: added the example and link to doc

File: site/content/blogs/release-v125.en.md [hunk 2]
```diff
@@ -33,6 +33,59 @@ Copilot v1.25 brings several new features and improvements:
 ## Environment Addons
 ### Import Values From CloudFormation Stacks In Workload Manifests
 
+You can now import values from environment addons CloudFormation stack or any other arbitary stack in your workload manifest using `from_cfn`.
```
Comment (huanjani) [inline site/content/blogs/release-v125.en.md]:
```suggestion
You can now import values from environment addons' CloudFormation stacks or any other stack in your workload manifest using `from_cfn`.
```

File: site/content/blogs/release-v125.en.md [hunk 3]
```diff
@@ -33,6 +33,59 @@ Copilot v1.25 brings several new features and improvements:
...
+To reference a value from another CloudFormation stack, users should first export the output value from the source stack.
+
+Here is an example on how the `Outputs` section of ClouFormation template looks like to export values from other stacks or to create a cross-stack references.
```
Comment (huanjani) [inline site/content/blogs/release-v125.en.md]:
```suggestion
Here is an example on how the `Outputs` section of a CloudFormation template looks when exporting values from other stacks or creating cross-stack references.
```

File: site/content/blogs/release-v125.en.md [hunk 4]
```diff
@@ -33,6 +33,59 @@ Copilot v1.25 brings several new features and improvements:
...
+```yaml
+Outputs:
+  WebBucketURL:
+    Description: URL for the website bucket
+    Value: !GetAtt WebBucket.WebsiteURL
+    Export:
+      Name: stack-WebsiteUrl # <- Unique export name within the region.
+```
+
+To find our more, you can see [this page](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/outputs-section-structure.html)
```
Comment (huanjani) [inline site/content/blogs/release-v125.en.md]:
```suggestion
To find our more, see [this page].(https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/outputs-section-structure.html) 
```

File: site/content/blogs/release-v125.en.md [hunk 5]
```diff
@@ -33,6 +33,59 @@ Copilot v1.25 brings several new features and improvements:
 ## Environment Addons
 ### Import Values From CloudFormation Stacks In Workload Manifests
 
+You can now import values from environment addons' CloudFormation stacks or any other stack in your workload manifest using `from_cfn`.
+To reference a value from another CloudFormation stack, users should first export the output value from the source stack.
+
+Here is an example on how the `Outputs` section of a CloudFormation template looks when exporting values from other stacks or creating cross-stack references.
```
Comment (huanjani) [inline site/content/blogs/release-v125.en.md]:
```suggestion
Here is an example of how the `Outputs` section of a CloudFormation template looks when exporting values from other stacks or creating cross-stack references.
```

**Review by huanjani — APPROVED** [review]
Thanks for addressing all the feedback. Just one more tiny nit. Feel free to remove the DNM label when you've addressed it.

**Pull request merged**